packages/affiliates/gas-rebate/VoterGasRebate.js

```diff
@@ -629,7 +629,8 @@ async function getHistoricalUmaEthPrice(dailyPrices) {
   for (let i = 0; i < dailyPrices.length; i++) {
     dailyUmaPrices.push({
       timestamp: dailyPrices[i].timestamp,
-      avgPx: umaEthPrices[i].toString(),
+      avgPx: umaEthPrices[i].toString().substr(0, 18), // Cut off any digits beyond 18th decimal so that we we can parse
```

Comment:
not a huge deal, but you can use `umaEthPrices[i].toFixed(18)` as well if this is a number and not a BN or string

Reply:
Aha! thanks